src/components/Genomes/Cobs/index.tsx

```diff
@@ -98,6 +101,60 @@ const CobsSearch: React.FC<CobsProps> = ({ catalogueName, catalogueID }) => {
       progress: undefined,
     });
 
+  const hypenateString = (str: string): string => {
+    const lowerCaseStr = str.toLowerCase();
+    const replacedSpaces = lowerCaseStr.replace(/ /g, '-');
+    return replacedSpaces.replace(/\./g, '-');
+  };
+
+  const getAccessionFromFirstGenome = async (): Promise<string> => {
+    const genomeName = hypenateString(catalogueName as string);
+    const url = `${config.api}genome-catalogues/${genomeName}/genomes?page=1&ordering=accession&page_size=1`;
+
+    try {
+      const response = await axios.get(url);
+      return response.data.data[0].attributes.accession;
+    } catch (responseErrors) {
+      return '';
+    }
+  };
+
+  const getSequenceCharsFromFastaFile = async (
+    fastaUrl: string
+  ): Promise<string> => {
+    try {
+      const headersConfig = {
+        headers: {
+          Range: 'bytes=0-499', // Fixed Range header value
+        },
+      };
+
+      const response = await axios.get(fastaUrl, headersConfig);
+      // Extract the DNA sequence from the data and get the first 50 characters
+      const { data } = response;
+      const lines = data.split('\n');
+      if (lines.length >= 2) {
+        const sequence = lines[1];
+        return sequence.substring(0, 50);
```

Comment:
Why only 50 here rather than (up to) the 500 we should have available? I guess this is just chopping off a potential FASTA header line right?

Reply:
@SandyRogers this has been adjusted